src/Avalonia.Themes.Default/ContextMenu.xaml

```diff
@@ -10,7 +10,7 @@
               BorderBrush="{TemplateBinding BorderBrush}"
               BorderThickness="{TemplateBinding BorderThickness}"
               Padding="{TemplateBinding Padding}">
-          <ScrollViewer>
+          <ScrollViewer Classes="menuscroller">
```

Comment:
We probably need some naming convention for built-in classes to avoid clashes with the user code